bats_test/help/no_command.bats

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bats
+
+@test "'ardrive' with no input arguments prints general help with all expected commands" {
+    run -0 yarn ardrive
+    [[ "$output" = "Usage: ardrive [command] [command-specific options]
```

Comment:
Let's factor the expected help output out to a variable to make this test more easily maintainable.